pkgs/by-name/sl/slob/package.nix

```
@@ -0,0 +1,5 @@
+{
+  python3,
+}:
+
+python3.pkgs.toPythonApplication python3.pkgs.slob
```

```suggestion
python3Packages.toPythonApplication python3Packages.slob
```

https://github.com/NixOS/nixpkgs/issues/211340